docs/releases/3.1.txt

```diff
@@ -90,7 +90,7 @@ Minor features
 * The default iteration count for the PBKDF2 password hasher is increased from
   180,000 to 216,000.
 
-* Added the :setting:`PASSWORD_RESET_TIMEOUT` setting to define the minimum
+* Added the :setting:`PASSWORD_RESET_TIMEOUT` setting to define the maximum
```

Comment:
I think we can chop `minimum`. Can you fix this also in other places? (`django/conf/global_settings.py` and `docs/ref/settings.txt`)

Reply:
Good call, and yep will do!